src/shared/components/Scrollbar/styles.module.scss

```diff
@@ -1,9 +1,9 @@
 .scrollbar {
-  width:90%;
+  width:100%;
```

Comment:
can you put a comment here about why this was necessary (so i can check it in browser)

Reply:
It just looked better.